moped-editor/src/views/projects/projectView/ProjectComponentEdit.js

```diff
@@ -694,10 +694,12 @@ const ProjectComponentEdit = ({
                 <Autocomplete
                   id="moped-project-subtype-select"
                   className={classes.formSelect}
-                  value={availableSubtypes.find(
-                    subtype =>
-                      subtype.toLowerCase() === selectedComponentSubtype
-                  )}
+                  value={
+                    availableSubtypes.find(
+                      subtype =>
+                        subtype.toLowerCase() === selectedComponentSubtype
+                    ) ?? null
+                  }
```

Comment:
This was the real fix for subtype, for some reason the value stays the same if `.find` returns `undefined`, but forcing it to return null instead clears the field for good.

Reply:
interesting, it wasnt triggering a refresh if it was undefined? in any case, thank you for the explanations!